BuildResidentialHPXML/measure.rb

```diff
@@ -493,6 +500,12 @@ def arguments(model)
     arg.setDefaultValue(Constants.Auto)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('rim_joist_assembly_r', false)
```

Comment:
Why is this argument not required but the other one is?

Reply:
I was following `foundation_wall_assembly_r`, but I suppose I shouldn't have been. I'll make it required, and default to 23.